runtime/queries/jsonc/highlights.scm

```diff
@@ -0,0 +1,22 @@
+[
+  (true)
+  (false)
+] @constant.builtin.boolean
+(null) @constant.builtin
+(number) @constant.numeric
+(pair
+  key: (_) @keyword)
+
+(string) @string
+(escape_sequence) @constant.character.escape
+(ERROR) @error
+
+"," @punctuation.delimiter
+[
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
```

Comment:
You can replace this part with `; inherits: json` to re-use `runtime/queries/json/highlights.scm`